README.md

```diff
@@ -190,4 +190,4 @@ Pull requests are welcome. For major changes, please open an issue first to disc
 Please make sure to update tests as appropriate.
 
 ## License
-[MIT](https://choosealicense.com/licenses/mit/)
+[Apache](https://choosealicense.com/licenses/apache-2.0/)
```

Comment:
Are all the commands in the README up to date with respect to the `panther-cli` commands?

Reply:
Yes, the `test` and `zip` commands still work the same way.